libraries/GCS_MAVLink/GCS.h

```diff
@@ -298,7 +298,7 @@ class GCS_MAVLINK
     MAV_RESULT handle_command_mag_cal(const mavlink_command_long_t &packet);
     MAV_RESULT handle_command_long_message(mavlink_command_long_t &packet);
     MAV_RESULT handle_command_camera(const mavlink_command_long_t &packet);
-
+    MAV_RESULT handle_command_do_send_banner(const mavlink_command_long_t &packet);
     // vehicle-overridable message send function
```

Comment:
Nitpicking: put an empty line before the comment

Reply:
Fixed.